app/scripts/modules/kubernetes/serverGroup/configure/wizard/volumes.html

```diff
@@ -116,6 +116,30 @@
         <input type="text" class="form-control input-sm" name="path" ng-model="source.secret.secretName"/>
       </div>
     </div>
+    <div class="form-group" ng-show="source.type === 'AWSELASTICBLOCKSTORE'">
+      <div class="col-md-3 sm-label-right">
+        Volume ID
+      </div>
+      <div class="col-md-4">
+        <input type="text" class="form-control input-sm" name="path" ng-model="source.awsElasticBlockStore.volumeId"/>
+      </div>
+    </div>
+    <div class="form-group" ng-show="source.type === 'AWSELASTICBLOCKSTORE'">
+      <div class="col-md-3 sm-label-right">
+        FS Type
```

Comment:
Consider adding help text to these fields (just a thought - not sure how obvious they are).